cmd/tracee-rules/output_test.go

```diff
@@ -112,26 +113,30 @@ HostName: foobar.local
 	}
 
 	for _, tc := range testCases {
-		var actualOutput bytes.Buffer
-		findingCh, err := setupOutput(&actualOutput, "", "", "", tc.outputFormat)
-		require.NoError(t, err, tc.name)
+		t.Run(tc.name, func(t *testing.T) {
+			actualOutput := NewSyncBuffer([]byte{})
+			findingCh, err := setupOutput(actualOutput, "", "", "", tc.outputFormat)
+			require.NoError(t, err, tc.name)
 
-		sm, _ := fakeSignature{}.GetMetadata()
-		findingCh <- detect.Finding{
-			Data: map[string]interface{}{
-				"foo1": "bar1, baz1",
-				"foo2": []string{"bar2", "baz2"},
-			},
-			Event:       tc.inputEvent,
-			SigMetadata: sm,
-		}
+			sm, err := fakeSignature{}.GetMetadata()
+			require.NoError(t, err)
+
+			findingCh <- detect.Finding{
+				Data: map[string]interface{}{
+					"foo1": "bar1, baz1",
+					"foo2": []string{"bar2", "baz2"},
+				},
+				Event:       tc.inputEvent,
+				SigMetadata: sm,
+			}
 
-		time.Sleep(time.Millisecond)
-		checkOutput(t, tc.name, actualOutput, tc.expectedOutput)
+			time.Sleep(time.Millisecond)
```

Comment:
We should get rid of this sleepy code one day, but I keep it out of scope to enable -race flag ASAP.